pkg/istio-agent/sds-agent.go

```diff
@@ -170,10 +167,6 @@ func NewAgent(proxyConfig *mesh.ProxyConfig, cfg *AgentConfig, sopts *security.O
 	//   the short lived certs.
 	// - if a JWTPath token exists, will be included in the request.
```

Comment:
Update the comment as well - and please add some info on why this is removed. If the JWT doesn't exist - it should not be set, so not clear removing this is desirable